plutus-playground/plutus-playground-client/src/Editor.purs

```diff
@@ -59,6 +60,7 @@ editorPane state =
         ]
     , br_
     , errorList
+, warningList
```

Comment:
I hear sirens. I think it's the indentation police. Quick, fix this before they get you!